router/router.go

```diff
@@ -583,6 +583,8 @@ func (worker *workerT) workerProcess() {
 }
 
 func (worker *workerT) processDestinationJobs() {
+	// TODO - this is the only place handleWorkerDestinationJobs is being called
```

Comment:
Either perform such a refactoring, if necessary, or be silent and remove the TODO 😄

Reply:
I am going to add another commit to inline this code